### PR TITLE
JHPIEGO - new expression

### PR DESCRIPTION
**Pull request by lwyszomi**
@snopoke 
http://manage.dimagi.com/default.asp?248939#1304050

### CODE REVIEW

File: custom/eqa/expressions.py [hunk 1]
```diff
@@ -36,27 +36,33 @@ def get_yes_no(val):
         return 'N/A'
 
 
+def get_two_last_forms(item, xmlns):
```
Comment (snopoke) [inline custom/eqa/expressions.py]: This is quite expensive, should it be cached?

**Comment by lwyszomi** [comment]
@snopoke updated

**Comment by snopoke** [comment]
build timed out - kicked it

**Comment by lwyszomi** [comment]
@snopoke 
Could you kick once again :)

**Pull request merged**